0016-pre-shared-key/0016-pre-shared-key.md

```diff
@@ -14,6 +14,28 @@ An advantage of PSK is that the pre-shared secret key only needs to be shared on
 
 A disadvantage of PSK is that it is repudiable. Although the sender does get cryptographic proof that the recipient received the payment, that proof cannot be used to convince 3rd parties that the sender did indeed send the funds, because the sender could have generated the fulfillment themselves. However, simple proof for the sender that the recipient got the funds is sufficient in many applications. The [Interledger Payment Request](https://github.com/interledger/rfcs/blob/master/0011-interledger-payment-request/0011-interledger-payment-request.md) transport protocol should be used instead in cases where non-repudiable proof is required.
 
+## Importance of PSK in ensuring network health
```

Comment:
This is the role of the Transport Layer, not just PSK. I don't think it belongs in this spec